lib/services/table/tableservice.js

```diff
@@ -748,7 +748,7 @@ TableService.prototype.deleteTableIfExists = function (table, optionsOrCallback,
 * @param {TableService~propertyResolver}  [options.propertyResolver]               The property resolver. Given the partition key, row key, property name, property value,
 *                                                                                  and the property Edm type if given by the service, returns the Edm type of the property.
 * @param {TableService~queryResponse} callback                                     `error` will contain information if an error occurs; 
-*                                                                                  otherwise `entities` will contain the entities returned by the query. 
+*                                                                                  otherwise `entries` will contain the entities returned by the query. 
 *                                                                                  If more matching entities exist, and could not be returned,
 *                                                                                  `queryResultContinuation` will contain a continuation token that can be used
```

Comment:
Thanks for the fix, This should be `continuationToken` as well, would you help to optimize it in the same PR? Thanks!